packages/hoppscotch-sh-admin/src/pages/users/index.vue

```diff
@@ -277,15 +276,18 @@ import IconX from '~icons/lucide/x';
 const t = useI18n();
 const toast = useToast();
 
+// Time and Date Helpers
 const getCreatedDate = (date: string) => format(new Date(date), 'dd-MM-yyyy');
 const getCreatedTime = (date: string) => format(new Date(date), 'hh:mm a');
+const lastActiveOn = (date: string | null) =>
+  date ? useTimeAgo(date).value : '';
```

Comment:
Shouldn't we keep a fallback copy instead of leaving it empty? Could you get it confirmed?
Also, we could rename the function to `getLastActiveOn` as per the convention followed.